From 334a936ea36d815da75287d09c432827c5bd4868 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Thu, 9 Aug 2018 18:18:59 +0300
Subject: [PATCH 0678/1051] fix: dts: a7k: Enable AP serial port on all A7K
 boards

Add missing enablement statement for AP UART0 on A7K board
flavors B, C, D and E.

Change-Id: Ie0569dcb00e60d7caeaa0d3909caf09271d017a6
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-B.dts | 4 ++++
 arch/arm64/boot/dts/marvell/armada-7040-db-C.dts | 4 ++++
 arch/arm64/boot/dts/marvell/armada-7040-db-D.dts | 4 ++++
 arch/arm64/boot/dts/marvell/armada-7040-db-E.dts | 4 ++++
 4 files changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
index 2929c23ad8e2..0e40e10993ed 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
@@ -22,6 +22,10 @@
 	};
 };
 
+&uart0 {
+	status = "okay";
+};
+
 &ap_sdhci0 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
index 8ca4a8ac32a7..b3670d00a250 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
@@ -26,6 +26,10 @@
 	};
 };
 
+&uart0 {
+	status = "okay";
+};
+
 &ap_sdhci0 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
index b4d5696a50d5..45c2602da9ea 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
@@ -26,6 +26,10 @@
 	};
 };
 
+&uart0 {
+	status = "okay";
+};
+
 &ap_sdhci0 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
index 53e8ae09708f..14af17dbd11f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
@@ -22,6 +22,10 @@
 	};
 };
 
+&uart0 {
+	status = "okay";
+};
+
 &cp0_i2c0 {
 	status = "okay";
 	clock-frequency = <100000>;
-- 
2.17.1

